awscli/customizations/emr/emrutils.py

```diff
@@ -153,20 +154,13 @@ def check_empty_string_list(name, value):
         raise exceptions.EmptyListError(param=name)
 
 
-def call(session, operation_object, parameters, region_name=None,
+def call(session, operation, parameters, region_name=None,
```

Comment:
In many places in this PR we've moved from `operation` to `operation_model`. Going from `operation_object` to `operation` feels backwards to me. Can we name this `operation_name` instead for consistency?